i18n/index.js

```diff
@@ -1,34 +0,0 @@
-import path from 'path';
-import LanguageDetector from 'i18next-browser-languagedetector';
-import NextI18Next from 'next-i18next';
-import { initReactI18next } from 'react-i18next';
-import getConfig from '../tenant.config';
-
-const config = getConfig();
-
-export default new NextI18Next({
```

Comment:
I still found some `import i18next` in our codebase. Shouldn't they all be removed?